auslib/test/test_db.py

```diff
@@ -2052,6 +2326,18 @@ def testUpdateReleaseChangeReadOnly(self):
     def testUpdateReleaseNoPermissionToSetReadOnly(self):
         self.assertRaises(PermissionDeniedError, self.releases.update, {"name": "c"}, {"read_only": True}, "bob", 1)
 
+    def testUpdateReleaseWithRuleMappingThatRequiresSignoff(self):
+        newBlob = ReleaseBlobV1(name="h", schema_version=1, hashFunction="sha256")
```

Comment:
No good reason, it's pretty much just an arbitrary string. In some of these tests, I'm explicitly changing the current hashFunction because it seems silly to have a scheduled change (even in tests) that doesn't actually change anything.